docs/sphinx/examples/cpp/basics/noise_amplitude_damping.cpp

```diff
@@ -20,12 +20,12 @@ int main() {
   // our damping channel to.
   cudaq::noise_model noise;
 
-  // Amplitude damping channel with `1.0` probability of the qubit
-  // decaying to the ground state.
+  // We define an amplitude damping channel setting to `1.0` the probability of 
+  // the qubit decaying to the ground state.
```

Comment:
```suggestion
  // We define an amplitude damping channel setting the probability of 
  // the qubit decaying to the ground state to `1.0`.
```
Reads better, I think.